src/main/target/MATEKF722SE_8MOTOR/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+target_stm32f722xe(MATEKF722SE_8MOTOR)
```

Comment:
@whirlybloke this can be done as a sub-target with some #ifdef on the target.c
See how OMNIBUSF4 does that for example. As a result it will be a 3-liner. Ok, maybe 5-liner

Reply:
Done. Removed MATEKF722SE_8MOTOR and mads it a sub target of MATEKF722SE